### PR TITLE
Doc review

### PR DESCRIPTION
**Pull request by Spomky**
Documentation reviewed.

This PR also takes into account:
- #64 
- #70

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,18 +52,60 @@ composer require spomky-labs/otphp --prefer-source
...
+echo 'The current OTP is: '.$otp->now();
+```
+
+In the example above, we use the `TOTP` class, but you can use the `HOTP` one the same way.
+
+Then, you have to configure you applications. 
+You can use the provisioning Uri (`$otp->getProvisioningUri();`) as QR Code input to easily configure all of them.
+
+We recommend you to use your own QR Code generator (e.g. [BaconQrCode](https://packagist.org/packages/bacon/bacon-qr-code)).
+If you do not have your own generator, the classes provide a convenient way to get an Uri to the Google Chart API which wil generate it for you:
```
Comment (bauerj) [inline README.md]: *will

File: doc/Customize.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
+# OTP Customization
+
+To generate ont-time passwords, each class needs at least the following parameters:
```
Comment (bauerj) [inline doc/Customize.md]: *one-time

File: doc/Factory.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# The Factory
+
+In some cases, you want to load a provisioning URI and the corresponding an OTP object.
```
Comment (bauerj) [inline doc/Factory.md]: > and the corresponding an OTP object

Should probably be 'and the corresponding OTP object'

**Comment by Spomky** [comment]
Many thanks @bauerj!

File: README.md [hunk 4]
```diff
@@ -76,13 +76,13 @@ $otp = TOTP::create();
 echo 'The current OTP is: '.$otp->now();
 ```
 
-In the example above, we use the `TOTP` class, but you can use the `HOTP` one the same way.
+In the example above, we use the `TOTP` class, but you can use the `HOTP` on the same way.
```
Comment (bauerj) [inline README.md]: Sorry to nit-pick but I think the previous version was correct.

**Pull request merged**